src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
 spring.application.name=hng-java-boilerplate
+
+# datasource configuration
+spring.datasource.url=jdbc:postgresql://localhost/{database}
```

Comment:
This whole url should be an environment variable. We don't need `localhost` here.

Reply:
Okay. I will effect the changes now